Casbin.Persist.Adapter.EFCore/EFCoreAdapter.cs

```diff
@@ -119,6 +136,15 @@ public virtual async Task AddPolicyAsync(string section, string policyType, IPol
             {
                 return;
             }
+
+            var filter = new PolicyFilter(policyType, 0, values);
+            var persistPolicies = filter.Apply(PersistPolicies);
```

Comment:
If `AutoSave` is false, Casbin will not call the method, why add filter here?

Reply:
if set autosave to true， it will add duplicate record， so i add filter here to